app/src/main/res/layout/dashboard_list_item_tools.xml

```diff
@@ -0,0 +1,130 @@
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="160dp"
+        android:padding="5dp">
+
+        <androidx.cardview.widget.CardView
+            android:layout_width="match_parent"
+            android:layout_height="150dp"
+            android:layout_margin="8dp"
+            android:elevation="@dimen/card_elevation"
+            android:onClick="@{() ->callbacks.onToolInfo(tool.code)}"
+            app:cardCornerRadius="@dimen/card_corner_radius">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+                <org.cru.godtools.base.ui.view.DaggerPicassoImageView
+                    android:layout_width="match_parent"
+                    android:layout_height="0dp"
+                    android:scaleType="centerCrop"
+                    android:src="@{banner}"
+                    app:layout_constraintBottom_toTopOf="@+id/guideline2"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    tools:srcCompat="@tools:sample/backgrounds/scenic" />
+
+                <ProgressBar
+                    style="@style/Widget.GodTools.Tools.Download.ProgressBar"
+                    android:layout_width="match_parent"
+                    android:layout_height="10dp"
+                    android:progress="@{downloadProgress}"
+                    android:visibility="gone"
+                    app:layout_constraintBottom_toTopOf="@id/guideline2"
+                    tools:max="12"
+                    tools:progress="5"
+                    tools:visibility="visible" />
+
+                <androidx.cardview.widget.CardView
+                    android:layout_width="28dp"
+                    android:layout_height="28dp"
+                    android:layout_marginTop="11dp"
+                    android:layout_marginEnd="11dp"
+                    android:background="@color/white"
+                    android:onClick="@{() -> tool.isAdded ? callbacks.removeTool(tool, primaryTranslation ?? parallelTranslation) : callbacks.addTool(tool.code)}"
+                    app:cardCornerRadius="40dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintTop_toTopOf="parent">
+
+                    <androidx.appcompat.widget.AppCompatImageView
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:padding="3dp"
+                        android:tint="@color/gt_blue"
+                        app:selected="@{tool.isAdded}"
+                        app:srcCompat="@drawable/ic_favorite" />
+                    <!--                 />-->
+
+                </androidx.cardview.widget.CardView>
```

Comment:
don't use a `CardView` for the favorites icon, just use an image with elevation.

you might be able to use a `ShapeableImageView` with a white background and the foreground being the current favorite icons.

Reply:
`ShapeableImageView` would involve some kotlin code. a different approach that should work instead is to create a simple xml shape drawable, and set this as the background image directly